src/main/java/de/hpi/bpt/argos/util/performance/NullWatchTask.java

```diff
@@ -0,0 +1,99 @@
+package de.hpi.bpt.argos.util.performance;
+
+/**
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AJavadoc_Style_BP2016)